css/style.css

```diff
@@ -44,42 +44,41 @@ a:hover {
   cursor: pointer;
 }
 main {
-  height: calc(100vh - 80px);
+  /* height: calc(100vh - 80px); */
+  font-weight: 300;
 }
-.s1Container {
-  padding-top: 10rem;
-  text-align: center;
+.fxdc-jcc-aic {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+button {
+  background-color: var(--primary-color);
+  border: transparent;
+  border-radius: 0.5rem;
+  color: #fff;
+  font-weight: 400;
+  margin-top: 2rem;
+  padding: 1rem 2rem;
 }
```

Comment:
It may be better to make a class for your button, so you wouldn't be confused as this affect all the buttons in your site!